src/main/java/rx/observables/SwingObservable.java

```diff
@@ -213,6 +215,34 @@ public Boolean call(ItemEvent event) {
         });
     }
     
+    /**
+     * Creates an observable corresponding to property change events.
+     * 
+     * @param component
+     *            The component to register the observable for.
+     * @return Observable of property change events for the given component
+     */
+    public static Observable<PropertyChangeEvent> fromPropertyChangeEvents(Component component) {
+        return PropertyChangeEventSource.fromPropertyChangeEventsOf(component);
+    }
+    
+    /**
+     * Creates an observable corresponding to property change events filtered by property name.
+     * 
+     * @param component
+     *            The component to register the observable for.
+     * @param proprtyName
+     *            A property name to filter the property events on.
+     * @return Observable of property change events for the given component, filtered by the provided property name
+     */
+    public static Observable<PropertyChangeEvent> fromPropertyChangeEvents(Component component, final String propertyName) {
+        return fromPropertyChangeEvents(component).filter(new Func1<PropertyChangeEvent, Boolean>() {
```

Comment:
Please prepend this call with ```PropertyChangeEventSource``` like the previous method, so that we're consistent with most of the other methods in this class (I know we're a little inconsistent here.)